elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/parser/handlebars/HandlebarsHelper.java

```diff
@@ -0,0 +1,246 @@
+                        + EIGHT_SPACES + "this." + fieldName + " = " + fieldName + ";" + NEWLINE + FOUR_SPACES + "}";
+    }
+
+    /**
+     * Generate setter method for Dimension Type.
+     * @param dim Dimension Object
+     * @return setter method
+     */
+    public String generateDimSetterMethod(Dimension dim) {
+
+        String type = getDimensionType(dim);
+        String fieldName = dim.getName();
+        return "public void set" + capitalizeFirstLetter(fieldName) + "(" + type + SPACE + fieldName + ") {" + NEWLINE
+                        + EIGHT_SPACES + "this." + fieldName + " = " + fieldName + ";" + NEWLINE + FOUR_SPACES + "}";
+    }
+
+    /**
+     * Generate setter method for Join Type.
+     * @param join Join Object
+     * @return setter method
+     */
+    public String generateJoinSetterMethod(Join join) {
+
+        String type = capitalizeFirstLetter(join.getTo());
+        String fieldName = join.getName();
+        return "public void set" + capitalizeFirstLetter(fieldName) + "(" + type + SPACE + fieldName + ") {" + NEWLINE
+                        + EIGHT_SPACES + "this." + fieldName + " = " + fieldName + ";" + NEWLINE + FOUR_SPACES + "}";
+    }
+
+    /**
+     * Generate getter method.
+     * @param type Type of the field
+     * @param fieldName field Name
+     * @return getter method
+     */
+    public String generateGetterMethod(String type, String fieldName) {
+
+        return "public " + type + " get" + capitalizeFirstLetter(fieldName) + "() {" + NEWLINE + EIGHT_SPACES
+                        + "return " + fieldName + ";" + NEWLINE + FOUR_SPACES + "}";
+    }
+
+    /**
+     * Generate getter method for Dimension type.
+     * @param dim Dimension Object
+     * @return getter method
+     */
+    public String generateDimGetterMethod(Dimension dim) {
+
+        String type = getDimensionType(dim);
+        String fieldName = dim.getName();
+        return "public " + type + " get" + capitalizeFirstLetter(fieldName) + "() {" + NEWLINE + EIGHT_SPACES
+                        + "return " + fieldName + ";" + NEWLINE + FOUR_SPACES + "}";
+    }
+
+    /**
+     * Generate getter method for Join type.
+     * @param join Join Object
+     * @return getter method
+     */
+    public String generateJoinGetterMethod(Join join) {
+
+        String type = capitalizeFirstLetter(join.getTo());
+        String fieldName = join.getName();
+        return "public " + type + " get" + capitalizeFirstLetter(fieldName) + "() {" + NEWLINE + EIGHT_SPACES
+                        + "return " + fieldName + ";" + NEWLINE + FOUR_SPACES + "}";
+    }
+
+    // Security.java helper methods
+    /**
+     * Get the Class Name for Security.
+     * @param name Name of the security rule
+     * @return Capitalize First Letter of Each Word and remove spaces
+     */
+    public String getClassSuffix(String name) {
```

Comment:
All the security helpers can probably be removed since they are parsing the rules and not the roles.  Rules is phase 2.